### ISSUE TITLE
Rename Link from Smallest Common Multiple to Least Common Multiple

### ISSUE THREAD

**Issue by nzraad** [freeCodeCamp/freeCodeCamp#15338]
I had to look at the Wiki to finally understand what was meant. It is misleading because the link is least common multiple, and for you to direct the user to that website implies Smallest Common Multiple is Least Common Multiple.

https://www.freecodecamp.com/challenges/smallest-common-multiple

**Comment by raisedadead** [comment]
> It is misleading because the link is least common multiple, and for you to direct the user to that website implies Smallest Common Multiple is Least Common Multiple.

Smallest === Least (in context of arithmetic, only)

Anyways, we are happy to change this to avoid any confusion

Contributors need to update the instruction to

> Smallest (or Least) Common Multiple

like so

![image](https://user-images.githubusercontent.com/1884376/27083489-56eca798-5066-11e7-8eaf-ef81e426282b.png)


Please follow our [Guidelines for Contributing](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md).

**Comment by jhowerin** [comment]
I'm still on the first timer since my first pull request had a couple process related issues - so I updated and submitted a pull request -

**Comment by raisedadead** [comment]
@jhowerin Thanks for making that pull request and the hard work. I am sincerely sorry for misleading and wasting your time.

I just noticed that on our beta (beta.freecodecamp.com) `staging`, we have dropped the MDN and other help links in favor of a planned hints system.

Meaning the changes to this will be mostly overridden hence, these links and resources will not be available in the format that they are in the production website (freecodecamp.com)

Hence this issue can be left as it is for now. (while this is a small change, we would want it NOT to be lost when beta is up)
